proto/viam/service/slam/v1/slam.proto

```diff
@@ -36,13 +36,16 @@ service SLAMService {
     };
   }
 
+  // DEPRECATED
```

Comment:
I'd try using this to have deprecation code generate https://stackoverflow.com/a/43380742

Reply:
If this doesn't work we can leave as is

Reply:
Works great! Thanks for the suggestion.